### PR TITLE
add ext-ops.rst

### PR DESCRIPTION
**Pull request by TonyYHsieh**
add ext-ops document.

### CODE REVIEW

File: docs/ext-ops.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+.. doxygenfunction:: hipblasltExtAMax
+
+
+hipblasLtExt Operation Usage
+================================
+
+Introduction
+--------------
+
+hipBLASLt has extension opertion APIs which is independent to gemm operation with. These extensions support:
```
Comment (Serge45) [inline docs/ext-ops.rst]: typo: operation.

File: docs/ext-ops.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+1. hipblasltExtSoftmax
+
+2. hipblasltExtLayerNorm
+    | covert a 2D tensor by LayerNorm to generate a new 2D normalized tensor .
+    | it is a independ function which can just call and get result.
```
Comment (Serge45) [inline docs/ext-ops.rst]: typo: convert at L34, an independent at L35.

File: docs/ext-ops.rst [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    | sample code is in clients/samples/ext_op/sample_hipblaslt_ext_op_layernorm.cpp
+
+3. hipblasltExtAMax
+    | Abs Maximum value of a 2D tensor.
+    | it is a independ function which can just call and get result.
```
Comment (Serge45) [inline docs/ext-ops.rst]: typo: independent.

File: library/include/hipblaslt-ext-op.h [hunk 4]
```diff
@@ -80,7 +80,7 @@ extern "C" {
 
 
 /*! \ingroup library_module
- *  \brief Perform layernorm on given tensor.
+ *  \brief Perform 2-D layernorm on with srouce input tensor and result output tensor.
```
Comment (Serge45) [inline library/include/hipblaslt-ext-op.h]: typo: source.

File: docs/ext-ops.rst (hunk 1 above)
Comment (TonyYHsieh) [reply docs/ext-ops.rst]: done

File: docs/ext-ops.rst (hunk 2 above)
Comment (TonyYHsieh) [reply docs/ext-ops.rst]: done

File: docs/ext-ops.rst (hunk 3 above)
Comment (TonyYHsieh) [reply docs/ext-ops.rst]: done

File: library/include/hipblaslt-ext-op.h (hunk 4 above)
Comment (TonyYHsieh) [reply library/include/hipblaslt-ext-op.h]: done

**Review by Serge45 — APPROVED** [review]

**Review by jichangjichang — APPROVED** [review]
LGTM,
Please use squash merge for this PR

**Pull request merged**